R/archive.R

```diff
@@ -75,7 +75,8 @@
 #'   sliding computation at any given reference time point t is performed on
 #'   **data that would have been available as of t**. More details on `slide()`
 #'   are documented in the wrapper function `epix_slide()`.
-#' 
+#'
+#' @importFrom R6 R6Class
```

Comment:
Following discussion: dropping this produces a Note in the R CMD Check:
> Namespace in Imports field not imported from: ‘R6’
     All declared Imports should be used.

Seemingly, it thinks that we aren't using `R6` despite the `R6::R6Class()` call below. So it wants us to remove it from DESCRIPTION. I don't understand why this is happening though.